src/e2e/java/uk/gov/hmcts/reform/sscs/functional/ccd/PipDecisionNoticeFunctionalTest.java

```diff
@@ -41,79 +41,6 @@
-        };
-    }
-
-    @NamedParameters("noAwardNoAwardComparisons")
-    @SuppressWarnings("unused")
-    private Object[] noAwardNoAwardComparisons() {
-        return new Object[] {
-            new Object[] {"lower", "lower", false, true, false},
-            new Object[] {"lower", "same", false, true, false},
-            new Object[] {"same", "lower", false, true, false},
-            new Object[] {"same", "same", false, false, false},
-            new Object[] {"lower", "lower", false, true, true},
-            new Object[] {"lower", "same", false, true, true},
-            new Object[] {"same", "lower", false, true, true},
-            new Object[] {"same", "same", false, false, true}
-        };
-    }
-
-    @NamedParameters("noAwardStandardRateComparisons")
-    @SuppressWarnings("unused")
-    private Object[] noAwardStandardRateComparisons() {
-        return new Object[] {
-            new Object[] {"lower", "lower", false, true, false},
-            new Object[] {"lower", "same", false, true, false},
-            new Object[] {"lower", "higher", true, true, false},
-            new Object[] {"same", "lower", false, true, false},
-            new Object[] {"same", "same", false, false, false},
-            new Object[] {"same", "higher", true, true, false},
-            new Object[] {"lower", "lower", false, true, true},
-            new Object[] {"lower", "same", false, true, true},
-            new Object[] {"lower", "higher", true, true, true},
-            new Object[] {"same", "lower", false, true, true},
-            new Object[] {"same", "same", false, false, true},
-            new Object[] {"same", "higher", true, true, true}
-        };
-    }
-
-    @NamedParameters("noAwardEnhancedRateComparisons")
-    @SuppressWarnings("unused")
-    private Object[] noAwardEnhancedRateComparisons() {
-        return new Object[] {
-            new Object[] {"lower", "same", false, true, false},
-            new Object[] {"lower", "higher", true, true, false},
-            new Object[] {"same", "same", false, false, false},
-            new Object[] {"same", "higher", true, true, false},
-            new Object[] {"lower", "same", false, true, true},
-            new Object[] {"lower", "higher", true, true, true},
-            new Object[] {"same", "same", false, false, true},
-            new Object[] {"same", "higher", true, true, true}
-        };
-    }
-
-    @NamedParameters("standardRateComparisons")
-    @SuppressWarnings("unused")
-    private Object[] standardRateComparisons() {
-        return new Object[] {
-            new Object[] {"lower", false, true, false},
-            new Object[] {"same", false, false, false},
-            new Object[] {"higher", true, true, false},
-            new Object[] {"lower", false, true, true},
-            new Object[] {"same", false, false, true},
-            new Object[] {"higher", true, true, true}
-        };
-    }
-
     @NamedParameters("allowed")
```

Comment:
Can probably just pass this in as 2 strings or booleans rather than an object array.